embedchain/vectordb/chroma.py

```diff
@@ -231,7 +233,13 @@ def query(self, input_query: List[str], n_results: int, where: Dict[str, any], s
                 " embeddings, is used to retrieve an embedding from the database."
             ) from None
         results_formatted = self._format_result(result)
-        contents = [result[0].page_content for result in results_formatted]
+        contents = []
+        for result in results_formatted:
+            context = result[0].page_content
+            content_metadata = result[0].metadata
+            content_source = content_metadata["url"]
+            content_doc_id = content_metadata["doc_id"]
+            contents.append((context, content_source, content_doc_id))
         return contents
```

Comment:
nit: seems like there is inconsistency in the naming scheme. Can you please change this block of code to look something like this:

```python
        contexts = []
        for result in results_formatted:
            context = result[0].page_content
            metadata = result[0].metadata
            source = metadata["url"]
            doc_id = metadata["doc_id"]
            contents.append((context, source, doc_id))
        return contexts
```

Reply:
okay